internal/tasks/manifests.go

```diff
@@ -710,11 +710,11 @@ func (j *Janitor) doVulnerabilityCheck(account keppel.Account, repo keppel.Repos
 		return nil
 	}
 
-	layerBlobs, continueCheck, err := j.checkPreConditionsForClair(account, repo, manifest, vulnInfo)
+	layerBlobs, err := j.checkPreConditionsForClair(account, repo, manifest, vulnInfo)
 	if err != nil {
 		return err
 	}
-	if !continueCheck {
+	if layerBlobs == nil {
```

Comment:
What happens if we return true together with an empty list of blobs? In that case, this would be a behavior change. Can this case happen?

Reply:
🤔 so we should rather do something like:
```suggestion
	if len(layerBlobs) == 0 {
```

Reply:
fixed in #219